include/mata/nfa/delta.hh

```diff
@@ -121,64 +121,105 @@
     const_iterator find(const Symbol symbol) const { return super::find({ symbol, {} }); }
 
     /**
-     * Iterator over moves. It iterates over pairs (symbol, target state) for a current source state whose state post
-     *  we iterate.
+     * @brief Iterator over moves represented as @c Move instances.
+     *
+     * It iterates over pairs (symbol, target) for the given @c StatePost.
      */
-    struct moves_const_iterator {
```

Comment:
Btw, did we want to make state post with a member instead of inheritance from OrdVector?

Reply:
Yes. We will refactor `StatePost` to have a member `symbol_posts` of type `OrdVector` instead of having it inherit from `Ordvector` for more explicit code which does not utilize more advanced features of C++ (inheritance of classes). It might be easier for students to read and understand the code with members instead of inheritance.